src/components/CreateAccountPrompt/CreateAccountPrompt.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import './CreateAccountPrompt.scss'
+
+const CreateAccountPrompt = (props) => {
+  const { setShouldPromptLogin, setPopup } = props;
+  setPopup(true)
+
+  return <div className='CreateAccountPrompt'>
+    <p>Would you like to create an account?</p>
+    <Link to='/create-user' onClick={() => setShouldPromptLogin(false)}> YUSH</Link>
+    <button onClick={() => { setShouldPromptLogin(false); setPopup(false) }}>NAH</button>
```

Comment:
Thinking at some point before Wednesday we should make this button an <a> tag so that they look the same. Otherwise, this looks great!

Reply:
Definitely agree, there are many inconsistencies in the styling currently. I believe it is within the scope of this branch so I will make an issue for it.

Reply:
issue: #41